### PR TITLE
fix: 🐛 remove left padding for quiet number input

### PR DESCRIPTION
**Pull request by kmelkon**

### CODE REVIEW

**Review by victoriatomzik — APPROVED** [review]

**Comment by victoriatomzik** [comment]
gg wp

**Review by jesperwiner — APPROVED** [review]

File: src/Molecules/Input/Number/Number.tsx [hunk 1]
```diff
@@ -139,7 +135,7 @@ const Input = styled(NormalizedElements.Input).attrs(() => ({ type: 'text' }))<P
          font-size: 28px; 
          font-weight: bold;
          &:focus {
-           padding-left: ${p.theme.spacing.unit(p.leftAddon ? 8 : 2)}px;
+           padding-left: ${p.theme.spacing.unit(p.leftAddon ? 3 : 2)}px;
```
Comment (fredriklu) [inline src/Molecules/Input/Number/Number.tsx:138]: Is there any reason why `leftAddon` is still left here? Perhaps this constant could be named into something more appropriate?

File: src/Molecules/Input/Number/Number.tsx [hunk 2]
```diff
@@ -76,8 +76,7 @@ const AddonBox = styled(Flexbox)<{ position?: 'left' | 'right' }>`
   top: 0;
   height: 100%;
   z-index: 3;
-  ${(p) => (p.position === 'left' ? `left: ${p.theme.spacing.unit(2)}px;` : '')}
-  ${(p) => (p.position === 'right' ? `right: ${p.theme.spacing.unit(2)}px;` : '')}
+  ${(p) => (p.position === 'right' ? `right: ${p.theme.spacing.unit(0)}px;` : '')}
```
Comment (LifeofMichal) [inline src/Molecules/Input/Number/Number.tsx:79]: What's this about? Do you need to take away this right for the padding to work?

**Comment by victoriatomzik** [comment]
@kmelkon Should we do something about it?

**Comment by kmelkon** [comment]
> @kmelkon Should we do something about it?

completely forgot about this, I'll check it now

File: src/Molecules/Input/Number/Number.tsx (hunk 2 above)
Comment (kmelkon) [reply src/Molecules/Input/Number/Number.tsx:79]: It's so the right addOn is aligned with the bottom border when not focused. Forgot to mention that in the PR description.
image

File: src/Molecules/Input/Number/Number.tsx (hunk 1 above)
Comment (kmelkon) [reply src/Molecules/Input/Number/Number.tsx:138]: Will fix, you're correct, it's not needed when using `quiet` anymore

**Review by Vikingjonsson — APPROVED** [review]

**Pull request merged**